.github/workflows/test-build.yml

```diff
@@ -7,15 +7,23 @@ env:
 
 jobs:
   lint:
-    name: lint
+    name: Lint
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: golangci-lint
+      - name: Run golangci-lint
         uses: golangci/golangci-lint-action@v2.3.0
         with:
           args: --timeout=5m
 
+  tidy:
+    name: Tidy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run go tidy
+        run: make tidy && git diff --exit-code && test -z "$(git ls-files --exclude-standard --others)"
```

Comment:
Maybe tidy should be a part of the following `test-build` job. Tidy runs go mod. The version of go should be the same when running tidy, test and build to have consistent results.

Reply:
Also, it'd be helpful to print the diff at the end when it fails.

Reply:
Yes, that's better.  I've moved the job, and it now outputs dirty files to stderr.  It's still not super-obvious when they're just new files but for changes you get full diffs.  A missing vendor module shows up as in https://github.com/storageos/api-manager/runs/1664181588?check_suite_focus=true#step:4:567

The last lines are the missing files:
```
vendor/gomodules.xyz/jsonpatch/v2/LICENSE
vendor/gomodules.xyz/jsonpatch/v2/go.mod
vendor/gomodules.xyz/jsonpatch/v2/go.sum
vendor/gomodules.xyz/jsonpatch/v2/jsonpatch.go
```